### ISSUE TITLE
WebODM hanging up at chmod

### ISSUE THREAD

**Issue by AnthonyCoughlin** [OpenDroneMap/WebODM#86]
Good day folks,

See attached.  I've cloned the most recent version of WebODM (today), and when executing the `./webodm.sh start` WebODM hangs up when trying to use chmod and trying to access the WebODM folder, says the file does not exist.

@pierotofy, I'm afraid this might be a pathway issue again.  I've tried running each of the following when attempting to start with no luck:

`./webodm.sh start`
`MSYS_NO_PATHCONV=1 ./webodm.sh start`
`COMPOSE_CONVERT_WINDOWS_PATHS=1 ./webodm.sh start`

Thanks again for the help!

![webodm_chmod2](https://cloud.githubusercontent.com/assets/25381671/22621172/851160a8-eae2-11e6-8838-fcff867e5f2d.JPG)

![webodm_chmod](https://cloud.githubusercontent.com/assets/25381671/22621180/d6031100-eae2-11e6-816e-bca3a1cd7112.JPG)

**Comment by pierotofy** [comment]
@AnthonyCoughlin thanks for reporting the issue.

Would you be able to provide additional details about how you cloned the repository? Did you use:

`git clone https://github.com/OpenDroneMap/WebODM --config core.autocrlf=input`

?

Also note that after an update of the WebODM code, you'll need to run:

```
./webodm.sh update
```

That command will take care of rebuilding all docker images related to WebODM.

**Comment by AnthonyCoughlin** [comment]
@pierotofy, yes, I ran using `--config core.autocrl=input`.  Didn't realize I needed to run `./webodm.sh` when cloning new folders.  Got it running, thanks again.

Side note:  See attached below.  I received this warning wasn't sure if it was of any importance:

Thanks again for your help,
Anthony

![webodm_update_error](https://cloud.githubusercontent.com/assets/25381671/22621762/b1395588-eaf0-11e6-982a-8a8e94ad45b2.JPG)
